docs/resources/tutorials/core/deployment/user_topics_multicast/user_topics_multicast.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<dds>
+    <profiles xmlns="http://www.eprosima.com/XMLSchemas/fastRTPS_Profiles">
+        <!-- Profile name shall match topic name -->
+        <data_reader profile_name="/chatter">
```

Comment:
```suggestion
<dds xmlns="http://www.eprosima.com/XMLSchemas/fastRTPS_Profiles">
    <profiles>
        <!-- Profile name shall match topic name -->
        <data_reader profile_name="/chatter" is_default_profile="true">
```

Reply:
Partially applied. they point here is that this is per topic, so I did not make it default profile on purpose